x-pack/plugins/enterprise_search/common/__mocks__/initial_app_data.ts

```diff
@@ -32,6 +32,11 @@ export const DEFAULT_INITIAL_APP_DATA = {
     hasSearchEnginesAccess: false,
     hasWorkplaceSearchAccess: true,
   },
+  features: {
+    hasNativeConnectors: true,
```

Comment:
question: Do you think we might need a `hasConnectors` as well, in case we don't get Elasticsearch to manage the connectors indices in time for this? I expect we'll be able to complete that work soon, but you never know what could happen.

Reply:
possibly I had that as an open question. I'm hopping everything on this object is temporary and can eventually be removed.

But I can add a `hasConnectors` and gate that as well. I am in the process of adding `hasDefaultIngestPipeline` right now to resolve some errors I just found.